### DISCUSSION TITLE
Automatically delete oldest plot if the destination folder is full

### DISCUSSION THREAD

**Discussion by Laplaceeee** [swar/Swar-Chia-Plot-Manager#888] [Ideas]
Since the pool protocol is coming soon, and many people try to utilize their old plots while plotting new plots.
That would be nice to add a function to automatically delete the oldest plot file in the destination folder if the disk space is full before moving the new plot to the destination folder.

**Comment by mbabak74** [comment]
Would be nice, but just delete the amount of plots that you can plot each day.
or write a script that always deletes the oldest file when available diskspace is less than 100gb
